src/Responses/Assistant/AssistantFileListResponse.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenAI\Responses\Assistant;
+
+use OpenAI\Contracts\ResponseContract;
+use OpenAI\Contracts\ResponseHasMetaInformationContract;
+use OpenAI\Responses\Concerns\ArrayAccessible;
+use OpenAI\Responses\Concerns\HasMetaInformation;
+use OpenAI\Responses\Meta\MetaInformation;
+use OpenAI\Testing\Responses\Concerns\Fakeable;
+
+/**
+ * @implements ResponseContract<array{object: string, data: array<int, array{id: string, object: string, model: string, created_at: int, finished_at: ?int, fine_tuned_model: ?string, hyperparameters: array{n_epochs: int|string}, organization_id: string, result_files: array<int, string>, status: string, validation_file: ?string, training_file: string, trained_tokens: ?int}>, has_more: bool}>
+ */
+final class AssistantFileListResponse implements ResponseContract, ResponseHasMetaInformationContract
+{
+    /**
+     * @use ArrayAccessible<array{object: string, data: array<int, array{id: string, object: string, model: string, created_at: int, finished_at: ?int, fine_tuned_model: ?string, hyperparameters: array{n_epochs: int|string}, organization_id: string, result_files: array<int, string>, status: string, validation_file: ?string, training_file: string, trained_tokens: ?int}>, has_more: bool}>
+     */
+    use ArrayAccessible;
+
+    use Fakeable;
+    use HasMetaInformation;
+
+    /**
+     * @param  array<int, RetrieveJobResponse>  $data
+     */
+    private function __construct(
+        public readonly string $object,
+        public readonly array $data,
+        public readonly string $firstId,
+        public readonly string $lastId,
```

Comment:
```suggestion
        public readonly ?string $firstId,
        public readonly ?string $lastId,
```

These should be nullable, as for no files the null is returned.

![Screenshot from 2023-11-09 16-23-17](https://github.com/openai-php/client/assets/924196/3062032b-5f4b-455d-9f8d-fc15cada2dd3)

Reply:
Makes sense. Thank you!

Currently I am implementing the "Threads" endpoints. I will push them soon.

Reply:
Awesome, thank you for quick additions :clap: